.prettierignore

```diff
@@ -2,6 +2,7 @@ dist-build/
 dist-start/
 
 # managed by sku
+coverage/
```

Comment:
This portion of the file is actually programatically generated, it shouldn't be manually updated. It'll need to be updated in the `configure` script.

Reply:
Thanks for the heads up. I've updated the configure script and related tests now.